### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Ditiselit**
- [DEMO LINK](https://ditiselit.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://ditiselit.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,9 +5,46 @@
...
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="images-product"
+        >
+
+        <div class="card__content">
+          <h2 class="name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <h4 class="code">Product code: 195434</h4>
+
```
Comment (sTorba24) [inline src/index.html:25]:
```suggestion
          <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
          <h4 class="card__code">Product code: 195434</h4>

```

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,46 @@
...
+          <div class="rating">
```
Comment (sTorba24) [inline src/index.html:26]:
```suggestion
          <div class="card__rating">
```

Comment (sTorba24) [reply src/index.html:26]: actually you can use like you have, in BEM we can have block in block, but it's more element of card than separated block

File: src/index.html [hunk 3]
```diff
@@ -5,9 +5,46 @@
...
+            <div class="stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+
+            <h3 class="rating__reviews">Reviews: 5</h3>
```
Comment (sTorba24) [inline src/index.html:35]:
```suggestion
            <h3 class="card__reviews">Reviews: 5</h3>
```

File: src/index.html [hunk 4]
```diff
@@ -5,9 +5,46 @@
...
+          </div>
+
+          <div class="price">
+            <h3 class="price__title">Price:</h3>
+            <h2 class="price__current">$2,199</h2>
+          </div>
```
Comment (sTorba24) [inline src/index.html:41]: same here, you can leave so, but it's not independent block, we using it only in card, so better to make it as element of card

File: src/styles/index.scss [hunk 5]
```diff
@@ -1,3 +1,124 @@
...
+  --grey-color: rgb(97, 96, 112);
+  --white-color: rgb(255, 255, 255);
+}
+
+.body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
```
Comment (sTorba24) [inline src/styles/index.scss:18]: move it to `*` selector

File: src/styles/index.scss [hunk 6]
```diff
@@ -1,3 +1,124 @@
...
+
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  margin-bottom: 24px;
+
+  &__reviews {
+    font-weight: 400;
```
Comment (sTorba24) [inline src/styles/index.scss:69]: it's default value

File: src/styles/index.scss [hunk 7]
```diff
@@ -1,3 +1,124 @@
...
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
```
Comment (sTorba24) [inline src/styles/index.scss:86]: same, remove

File: src/styles/index.scss [hunk 8]
```diff
@@ -1,3 +1,124 @@
...
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: var(--dark-blue-color);
+    margin: 0;
+  }
+}
+
+.button {
+  box-sizing: border-box;
```
Comment (sTorba24) [inline src/styles/index.scss:102]: remove it after setting on `*`

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Almost good! Lets make it perfect

File: src/styles/index.scss (hunk 8 above)
Comment (Ditiselit) [reply src/styles/index.scss:102]: What? Write more clearly!

**Pull request closed without merging**